stryker-javascript-mutator fails to load in node 4

### Summary

Locally, stryker-javascript-mutator works for me with Node 8. With Node 4, it fails to load.

Running "stryker:sauceLabsRun" (stryker) task
[DEBUG] ConfigReader - [39mLoading config stryker.conf.js
[WARN] PluginLoader - [39mError during loading "stryker-javascript-mutator" plugin:
Cannot find module 'lodash'

### Stryker config

<!--- Please place your stryker config below. Feel free to change paths in the files and mutate arrays if you cannot share them. -->

I have some additional grunt config, but I don't think it's relevant.

```js
config.set({
        testRunner: 'karma',
        testFramework: 'jasmine',
        reporter: ['html', 'clear-text', 'event-recorder'],
        mutator: 'javascript',
        files: [
            {
                pattern: 'src/**/*.js',
                included: true,
                mutated: true
            },
            'test/**/!(*e2e)spec.js',
            {
                pattern: 'test/**/*.json',
                included: false,
                mutated: false
            }
        ],
        logLevel: 'info',
        thresholds: { high: 99, low: 94, break: 90 }
    });
}
```

### Stryker environment

<!-- Please list your stryker plugins + versions here (output of `npm ls | grep stryker`) -->

```
    "grunt-sonar-runner": "2.4.3",
    "stryker": "^0.17.0",
    "stryker-api": "^0.12.0",
    "stryker-html-reporter": "^0.11.4",
    "stryker-jasmine": "^0.7.3",
    "stryker-javascript-mutator": "^0.3.1",
    "stryker-karma-runner": "^0.11.3",
```

### Your Environment
| software         | version(s)
| ---------------- | -------  
| node             |  4.x
| npm              |  2.x
| Operating System |  RHEL or Windows

--- a/packages/stryker-javascript-mutator/package.json
+++ b/packages/stryker-javascript-mutator/package.json
@@ -40,6 +40,7 @@
     "babel-core": "^6.26.0",
     "babel-generator": "^6.26.0",
     "babylon": "^6.18.0",
+    "lodash": "^4.17.4",
     "log4js": "^1.1.1",
     "tslib": "^1.8.0"
   },

